tests/helpers/kubernetes/tunnel.py

```diff
@@ -20,7 +21,10 @@ def deploy_fake_backend_proxy_pod(namespace):
     """
     corev1 = kube_client.CoreV1Api()
     pod_yaml = Path(SCRIPT_DIR / "tunnel/pod.yaml").read_bytes()
-    pod = corev1.create_namespaced_pod(body=yaml.safe_load(pod_yaml), namespace=namespace)
+    pod = retry(
```

Comment:
Do you know why this was failing?  It seems more like a bug in Kubernetes if the pod fails to be created.